app/Main.hs

```diff
@@ -2,6 +2,7 @@
 {-# LANGUAGE TemplateHaskell #-}
 module Main where
 
+import Control.Monad (forM_)
```

Comment:
I usually try to group imports into two groups:

* imports coming from this packages
* all other imports